src/js/helpers/searchHelper.js

```diff
@@ -178,3 +247,38 @@ export const performFinancialSystemLookup = (params) => {
         }
     };
 };
+
+// Budget Filters consist of the three elements of Budget Categories filter
```

Comment:
Let's keep the SearchHelper dedicated to making API calls. You have a budgetCategoryHelper.... or you can make a new helper.